### DISCUSSION TITLE
Test Delete Route

### DISCUSSION THREAD

**Discussion by methos02** [orchidsoftware/platform#2530] [Q&A]
Hello, I try to make a test for delete permission route but I always have redirect status instaead of 401. 

There is my test, url is it good ?
`

    public function cant_other_element_delete():void {
        $element = Element::factory()->create();

        $res = $this->actingAs($this->createUser())->ajaxPost('/admin/elements/remove?id='. $element->id);
        $res->assertStatus(401);
    }
`

**Comment by tabuna** [comment]
Hi, most likely you are getting a 401 error because the user does not have enough [basic rights](https://github.com/orchidsoftware/platform/blob/ce2fb7a5c563329886f6ce3c1464f70ae581dfc5/src/Platform/Http/Middleware/Access.php#L89)

Try usage of factory user with all permissions: 

```php
use Orchid\Support\Facades\Dashboard;

$user = User::factory()->create([
  'permissions' => Dashboard::getAllowAllPermission(),
]);

$screen = $this->screen('platform.example')->actingAs($user);
```

**Comment by methos02** [comment]
Hello, thx for the answer :D 
I would like have 401 (to desable user to delete element from other user), but i have de 302 and don't know where it come from. 

(sorry for my english :'( )
